describealign.py

```diff
@@ -995,7 +996,7 @@ def read_config_file(config_path):
     write_config_file(config_path, settings)
   return settings
 
-def settings_gui(config_path):
+def settings_gui(config_path: Path):
```

Comment:
Why the type hint here specifically?

Reply:
This was to make it clear what type is being passed, as it used to be a string but with the new platformdirs call, its a Path object and some functions don't work the same (ie string concat)